meson.build

```diff
@@ -1,6 +1,4 @@
-project('lighttpd', 'c', version: '1.4.68', license: 'BSD-3-Clause',
-  default_options : ['c_std=c11']
-)
+project('lighttpd', 'c', version: '1.4.68', default_options : ['c_std=c11'], meson_version: '>=0.40.0')
```

Comment:
I liked the previous wrapping. but why is the license key silently removed? (not described in commit)

Reply:
Rebasing issue. Will fix.

Reply:
Fixed.